app/backend/app.py

```diff
@@ -83,10 +83,57 @@ async def assets(path):
     return await send_from_directory(Path(__file__).resolve().parent / "static" / "assets", path)
 
 
+# Decorator for routes that request a specific file that might require access control enforcement
```

Comment:
PEP8 nit: This comment here seems like it's basically a function docstring and should be inside the function. https://peps.python.org/pep-0257/

Not a big deal for top-level functions but a good practice generally for better intellisense/hover-ability.